--- Portfile.orig	2008-12-18 08:07:51.000000000 -0800
+++ Portfile	2008-12-18 08:36:18.000000000 -0800
@@ -1,7 +1,7 @@
 PortSystem 1.0
 
 name		          uclatools
-version           1.0.1
+version           1.0.3
 categories        science graphics
 maintainers       jameskyle@ucla.edu
 description       Set of utilities for converting, analysing, and manipulating MRI image data.
@@ -10,9 +10,9 @@
 homepage          http://www.ccn.ucla.edu/BMCweb/SharedCode/SharedSoftware.html
 platforms		      darwin
 master_sites		  http://www.ccn.ucla.edu/jkyle/
-checksums		      md5 9d3aa05f89fc4979bbfeccbb43c3624e \
-                  sha1 798387529b586f949eb52bbe7ba92440cbeb1c99 \
-                  rmd160 ccf0dd99d3886c6c1bbb55cbc277d5e2460ea0b6
+checksums		      md5 48246c595203f7c17bc1956d65ce58e5 \
+                  sha1 0e1c010ee5406ca2677569cc0588e8dd30cb7746 \
+                  rmd160 e57fed6bdaa6d85e2c75fde99f391f2c4c6d593a 
 
 depends_build port:cmake
 configure.cmd cmake
